source/agora/node/Ledger.d

```diff
@@ -189,8 +204,12 @@ public class Ledger
 
     private const(Output)* findOutput (Hash tx_hash, size_t index) @safe
     {
-        foreach (ref block; this.ledger)
+        Block block;
```

Comment:
Move the block declaration inside of the `foreach` loop.

Reply:
Applied the suggestion